src/utils/aws/ses.py

```diff
@@ -1,12 +1,21 @@
 """Create DNS email records from SES."""
 # Third-Party Libraries
 import boto3
+from botocore.exceptions import ClientError
+
+# cisagov Libraries
+from settings import APP_ENV, APP_NAME, SES_ASSUME_ROLE_ARN, SMTP_FROM
+from utils.aws import sts
 
 route53 = boto3.client("route53")
-ses = boto3.client("ses")
+
+if SES_ASSUME_ROLE_ARN:
+    ses = sts.assume_role_client("ses", SES_ASSUME_ROLE_ARN)
+else:
+    ses = boto3.client("ses")
```

Comment:
This is not going to work having it like this. We need to assume the SES role when sending email, but we would not want to use that assumed role when creating an email address for one of our domains.